packages/commerce-sdk-react/src/hooks/ShopperLogin/helper.test.tsx

```diff
@@ -7,16 +7,7 @@
 import React from 'react'
 import {useShopperLoginHelper} from './helper'
 import {fireEvent, waitFor, screen} from '@testing-library/react'
-import {renderWithProviders} from '../../test-utils'
-
-const mockLoginGuestUser = jest.fn().mockResolvedValue('mockLoginGuestUser')
-
-jest.mock('../useAuth', () => {
-    return jest.fn(() => ({
-        ready: () => Promise.resolve({access_token: '123'}),
-        loginGuestUser: mockLoginGuestUser
-    }))
-})
```

Comment:
It turns out that when we mocked `useAuth` like this, there was still some code that bypassed it and called `ready` from the Auth instance directly.

So a better module to mock is the `Auth` class:

```js
jest.mock('../../auth', () => {
    return class Auth {
        ready() {
            return Promise.resolve({access_token: '123'})
        }

        loginGuestUser() {
            return mockLoginGuestUser()
        }
    }
})
```